modules/account-lib/src/coin/hbar/transactionBuilder.ts

```diff
@@ -9,29 +11,33 @@ import {
   SigningError,
 } from '../baseCoin/errors';
 import { BaseAddress, BaseFee, BaseKey } from '../baseCoin/iface';
-import { proto } from '../../../resources/hbar/protobuf/hedera';
 import { Transaction } from './transaction';
-import { getCurrentTime, isValidAddress, isValidRawTransactionFormat, isValidTimeString, toUint8Array } from './utils';
+import {
+  getCurrentTime,
+  isValidAddress,
+  isValidRawTransactionFormat,
+  isValidTimeString,
+  toUint8Array,
+  toHex,
+} from './utils';
 import { KeyPair } from './keyPair';
-import { SignatureData, HederaNode } from './ifaces';
+import { SignatureData, HederaNode, Timestamp } from './ifaces';
 
 export const DEFAULT_M = 3;
 export abstract class TransactionBuilder extends BaseTransactionBuilder {
   protected _fee: BaseFee;
   private _transaction: Transaction;
   protected _source: BaseAddress;
-  protected _startTime: proto.ITimestamp;
+  protected _startTime: Timestamp;
   protected _memo: string;
-  protected _txBody: proto.TransactionBody;
   protected _node: HederaNode = { nodeId: '0.0.4' };
-  protected _duration: proto.Duration = new proto.Duration({ seconds: 120 });
+  protected _duration = 120;
```

Comment:
Move the 120 to DEFAULT_DURATION just like DEFAULT_M

Reply:
Sure